darts/models/forecasting/regression_model.py

```diff
@@ -1032,3 +1032,358 @@ def __init__(self):
 
     def __str__(self):
         return f"_QuantileModelContainer(quantiles={list(self.keys())})"
+
+
+class RegressionModelWithCategoricalCovariates(RegressionModel):
```

Comment:
What's the best place for this new class? Within regression_model.py (as it is now) or in the models/forecasting directory?

Reply:
looks good in this file 👍